uplink/storage/segments/store.go

```diff
@@ -151,8 +151,8 @@ func (s *segmentStore) Put(ctx context.Context, streamID storj.StreamID, data io
 	return Meta{}, nil
 }
 
-// Get requests the satellite to read a segment and downloaded the pieces from the storage nodes.
-func (s *segmentStore) Get(
+// Download downloadeds the pieces from the storage nodes.
```

Comment:
In fact its not downloading anything. This method is mainly creating `Ranger` for segment that will be later used to download object.

Reply:
Give me a better name then. I was more influenced by the second part that the method documentation had, considering that I only removed the request to the satellite.

Reply:
hmm I would say just `GetRanger` but I'm not sure what to do with second return value. Maybe `GetDownloadMetadata`?

Reply:
Taking a look now (sorry, I should have done that before renaming), I think that we can remove the 2 return parameter because it's coming from `info` parameter so it doesn't make sense to return it to the caller when the caller already has it.
Once that be done, I would rename it to `Ranger` to follow Go standard conventions regarding getter methods. Nonetheless, I'm fine of renaming to `GetRanger` is you prefer so.

What do you think?

Reply:
I like it. `Ranger` is also fine for me :+1: